src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashBuilds.java

```diff
@@ -49,7 +49,9 @@ public void onCompleted(AbstractBuild build, TaskListener listener) {
             return;
         }
         Result result = build.getResult();
-        JenkinsLocationConfiguration globalConfig = new JenkinsLocationConfiguration();
+        // Note: current code should no longer use "new JenkinsLocationConfiguration()"
```

Comment:
I would recommend removing this comment seems like it adds no value, the reasoning behind the change can be in the commit message or the PR

Reply:
Well, to many newcomers (like myself not so long ago) the code is the documentation. A removed line carries no visibility to `git blame`, not that many people dig in that at all, and a checked-out workspace or a fork has no data from PR discussions. For this particular piece of code, I found too many examples collecting dust on the internet over many years proposing the `new` way to get the config, that without the comment someone might be eager to make this particular mistake again.